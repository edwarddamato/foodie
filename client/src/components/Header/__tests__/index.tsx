import * as React from 'react';
import { Header } from '../index';
import { create } from 'react-test-renderer';

it('renders correctly', () => {
  const app = create(
    <Header />
  ).toJSON();
  expect(app).toMatchSnapshot();
});