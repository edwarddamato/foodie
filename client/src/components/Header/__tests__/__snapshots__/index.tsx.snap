// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<header
  className="component_header"
>
  <h1
    className="component_header-title"
  >
    foodie
  </h1>
</header>
`;
